src/llmq/quorums_utils.h

```diff
@@ -89,6 +89,22 @@ class CLLMQUtils
         }
         return HexStr(vBytes);
     }
+    template <typename CacheType>
+    static void QuorumsCacheInit(CacheType& cache)
+    {
+        for (auto& llmq : Params().GetConsensus().llmqs) {
+            cache.emplace(std::piecewise_construct, std::forward_as_tuple(llmq.first),
+                                                    std::forward_as_tuple(llmq.second.signingActiveQuorumCount));
```

Comment:
Hm, should still be `2*signingActiveQuorumCount`, no? If you look [here](https://github.com/dashpay/dash/blob/master/src/unordered_lru_cache.h#L24) `truncateThreshold` gets `2*maxSize` assigned in our case which is set to `signingActiveQuorumCount` in `Yooda()`. And the cache always seems to contain `truncateThreshold` entries if you look [here](https://github.com/dashpay/dash/blob/master/src/unordered_lru_cache.h#L90) and [here](https://github.com/dashpay/dash/blob/master/src/unordered_lru_cache.h#L104) (if i don't miss something) 🤔

Reply:
> the cache always seems to contain truncateThreshold 

Hmm, no... It is [truncated to `maxSize`](https://github.com/dashpay/dash/blob/master/src/unordered_lru_cache.h#L104-L106) whenever you [reach `truncateThreshold + 1` size](https://github.com/dashpay/dash/blob/master/src/unordered_lru_cache.h#L90-L92).

Reply:
Ahh damn right, seems like i then confused myself about that loop 😄 Hmm ok, soo either we do a `signingActiveQuorumCount*2` hardcoded or we add something like  `-llmq-cache-size-multiplier` with a default value of 2 so that it would be `signingActiveQuorumCount*2` by default but increasable?

Reply:
Well, I mean, we mostly care about `signingActiveQuorumCount+1` cause that's what we use for sig verification, so this should be a minimum imo. I'm trying to think about any situation where we would need even older quorums and it's probably just some rpc quorum commands that might be executed from time to time if someone wants to get an info about some old quorum or verify a signature which was created long time ago (not sure why would you want that though). But in other cases I _think_ we should be fine with `signingActiveQuorumCount+1`.

Reply:
>  I'm trying to think about any situation where we would need even older quorums and it's probably just some rpc 

Yeah agree, looked at it again and i also don't see any other reason than RPC for a deeper check than `signingActiveQuorumCount +1`. I set it to that for now, see 5856ccec5fd33947275374225a1796f26e489835. But theoretically such fun could be frequently triggered by platform if they for whatever reason need data from quorums older than we have cached.

@shumkov @cofresi @antouhou do you see any potentially reason in platform to query data from inactive quorums? We plan to reduce the cache to "the last 25 quorums for each of the last 25 blocks" for 100 member quorums.  Please let us know if thats not enough :)

Reply:
@xdustinface Well, for active quorums we can verify the sigs on our own - the RPC is _specifically_ to interact with non-active quorums, so we don't need to build a quorum cache on the platform

Reply:
> the RPC is _specifically_ to interact with non-active quorums, so we don't need to build a quorum cache in the platform

If i got it right, this means you need to query inactive quorums? Let's just discuss this in the meeting later!